### PR TITLE
feat: create WebSockets module

### PR DESCRIPTION
**Pull request by valerydluski**
#### Title of Pull Request

create websockets module

#### 🤔 This is a ...

- [ ] 🌟 New task
- [x] 🌐 New module
- [ ] ⚙️ Update to an existing task
- [ ] 🔧 Update to an existing module
- [ ] 🔗 Update or addition of external resources or links
- [ ] 🐛 Fix in a task or related content
- [ ] 🛠 Fix in a module or related content
- [ ] ✏️ Fixed a typo or grammatical error
- [ ] 🔗 Fixed a broken link
- [ ] ❓ Other (specify: **\*\*\*\***\_\_\_\_**\*\*\*\***)

#### Checklist

- [x] ✅ I have performed a self-review of my own code.
- [x] 📝 I have commented my code, particularly in hard-to-understand areas.
- [x] 🔧 I have made corresponding changes to the documentation (if applicable).
- [x] 🚫 My changes generate no new warnings or errors.

### CODE REVIEW

**Review by KristiBo — APPROVED** [review]
:thumbsup: thank you

**Review by stardustmeg — APPROVED** [review]

File: stage2/modules/web-sockets/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+   - [How to secure your WebSocket connections](https://www.freecodecamp.org/news/how-to-secure-your-websocket-connections-d0be0996c556/)
+
+4. **Lecture Videos:**
+   - [WebSockets [RU]](https://youtu.be/rpyk9JGAPZs)
+   - [Rolling Scopes School Lithuania. WebSocket](https://youtu.be/II77rlvVx-k)
+   - [Rolling Scopes School Georgia. WebSocket](https://youtu.be/kV5tOp7tdzM)
+
+## Practice 💻
+
+- Complete the "[St2] WebSockets" test in the RS-App > Auto Test.
```
Comment (haslie22) [inline stage2/modules/web-sockets/README.md:47]: Shouldn't the [Fun Chat](https://github.com/rolling-scopes-school/tasks/blob/master/stage2/tasks/fun-chat/README.md) task also be included here?

**Review by haslie22 — APPROVED** [review]
LGTM 👍

File: stage2/modules/web-sockets/README.md (hunk 1 above)
Comment (valerydluski) [reply stage2/modules/web-sockets/README.md:47]: nope

**Pull request merged**